packages/frontend-web/src/app/scenes/Settings/Settings.tsx

```diff
@@ -1118,24 +1054,28 @@ export class Settings extends React.Component<ISettingsProps, ISettingsState> {
 
     return (
       <div {...css(STYLES.base)}>
-        <HeaderBar logout={logout} homeLink title="Settings"/>
+        <HeaderBar logout={logout} homeLink title="Settings" />
         <div {...css(STYLES.body)}>
           <h1 {...css(VISUALLY_HIDDEN)}>Open Source Moderator Settings</h1>
-          {this.renderUsers()}
+          {/* TEMP COMMENTING OUT TO MAKE DEV EASIER */}
+          {/* {this.renderUsers()} */}
```

Comment:
?

Reply:
WOOPS, need to uncomment this and move the Restricted Terms component way down.